master-firmware/src/udp_topic_injector.c

```diff
@@ -0,0 +1,43 @@
+#include <ch.h>
+#include <lwip/api.h>
+#include "udp_topic_injector.h"
+#include "msgbus_protobuf.h"
+#include "main.h"
+
+static void udp_topic_injector_thd(void* p)
+{
+    (void)p;
+
+    struct netconn* conn;
+
+    chThdSetPriority(HIGHPRIO);
+    chRegSetThreadName(__FUNCTION__);
+
+    /* Listens to UDP on port 10'000 */
+    conn = netconn_new(NETCONN_UDP);
+    chDbgAssert(conn != NULL, "Cannot create a connection object");
+    netconn_bind(conn, IPADDR_ANY, 10000);
```

Comment:
port as param?

Reply:
That would also require watching the port for changes, and reclosing/reopening it if the port changes. Considered out of scope.

Reply:
OK